ambry-api/src/main/java/com/github/ambry/quota/storage/StorageQuotaService.java

```diff
@@ -49,6 +50,13 @@
    */
   boolean shouldThrottle(short accountId, short containerId, QuotaOperation op, long size);
```

Comment:
Same comment as for `shouldThrottle` method above.

Reply:
updated.